### ISSUE TITLE
Tags in sidebar doesn't display on /activity or /followers, /search/ to /tagged/ fix doesn't either.

### ISSUE THREAD

**Issue by exinspired** [new-xkit/XKit#810]
Most recent Chrome, most recent New Xkit, it's been happening for a while so I'm not sure how relevant those are at this point haha
(also I really hope these haven't been reported already.)

I'm not certain if that's intended or not but it never comes up for me, and the search bar tracking /search/->/tagged/ doesn't work on those pages either. I feel like it'd be nice if they could display there, though that may be because i sort of hub my tumblr usage at the activity page.  I can't help but wonder if any of this has anything to do with the same issue that I had earlier with the [quick tags css not displaying on /activity/](https://github.com/new-xkit/XKit/issues/655), since it's the same pages having the issue.

Also, while getting these screenshots, the /search/->/tagged/ thing would only switch it like two times out of three. which is good and still functional, but i only caught it because I was looking. But I have a feeling it's a loading speed thing rather than an issue on your ends.  (Also, while I was taking a screenshot it would always revert back to the /search/ link. Which was interesting, but snipping tool has a delay feature on it, so I caught it anyway--it may have been Greenshot's fault, though.)

Excuse the imageheaviness and sloppiness. Captions included in case you're not sure what you're looking at.

**Search bars and search bar tracking**
activity page showing 'Search Tumblr' instead of 'Search (new)'
![activity page showing 'Search Tumblr' instead of 'Search (new)'](http://i.imgur.com/Nn080Ed.png)
activity page showing tracked tags without any updates and linking to /search/ instead of displaying the updated tag and linking to /tagged/
![activity page showing tracked tags without any updates and linking to /search/ instead of displaying the updated tag and linking to /tagged/](http://i.imgur.com/XeMZP1x.png)

blog page showing 'Search (new)' instead of 'Search Tumblr'
![blog page showing 'Search (new)' instead of 'Search Tumblr'](http://i.imgur.com/iDPhnRl.png)
blog page showing tracked tags with update and linking to /tagged/ instead of not displaying updates and linking to /search/
![blog page showing tracked tags with update and linking to /tagged/ instead of not displaying updates and linking to /search/](http://i.imgur.com/fwfMuEb.png)

**Sidebar**
Sidebar on activity page displaying the old sidebar(Old Sidebar not installed) and nothing beneath it
![Sidebar on activity page displaying the old sidebar(Old Sidebar not installed) and nothing beneath it](http://i.imgur.com/mbjUEzV.png)
Sidebar on followers page(slightly scrolled down) showing the radar beneath the old sidebar(Old Sidebar not installed)
![Sidebar on followers page(slightly scrolled down) showing the radar beneath the old sidebar(Old Sidebar not installed)](http://i.imgur.com/RnrGvci.png)
Sidebar on the /blog/ page displaying the old sidebar and tracked tags
![Sidebar on the /blog/ page displaying the old sidebar and tracked tags](http://i.imgur.com/YMHv7JD.png)
Sidebar on dashboard displaying the recommended blogs list and tracked tags
![Sidebar on dashboard displaying the recommended blogs list and tracked tags](http://i.imgur.com/vuEXmBD.png)

**Comment by Wolvan** [comment]
Yeah, we know about that. I don't know why, but tag tracking+ has code to limit it to dashboard only

**Comment by exinspired** [comment]
Oh, okay! My bad, then--I should've figured, since it's been going so long
it's not news haha.
